webapp/src/components/modals/attach_comment_to_issue/attach_comment_to_issue.jsx

```diff
@@ -128,7 +128,6 @@ export default class AttachIssueModal extends PureComponent {
                 >
                     <Modal.Body
                         style={style.modalBody}
-                        ref='modalBody'
```

Comment:
There is no trace of this pattern in mattermost-webapp. I'm not sure what it's original purpose was, but removing it is not breaking any functionality regarding the modals.